### PR TITLE
Basic layout

### PR DESCRIPTION
**Pull request by Marisa5280**
### description
Created a webpage with a header and 5 boxes with corresponding hex codes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <!-- <link rel="preconnect" href="https://fonts.googleapis.com"> -->
```
Comment (CaliHam) [inline index.html]: Let's take comments out of the code before pushing it to main!

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
+* {
+    font-family: 'Poppins', sans-serif;
+    background-color: #E7F1F1;
+    /* border: black 2px solid */
```
Comment (CaliHam) [inline styles.css]: Here too!

**Review by CaliHam — CHANGES_REQUESTED** [review]
Let's take comments out of the code before pushing it to main!

**Review by Marisa5280 — COMMENTED** [review]
Removed commented lines!

**Review by CaliHam — APPROVED** [review]
Thank you! Looks great!

**Pull request merged**